resources/fetching/fetchers/azure/assets_fetcher.go

```diff
@@ -40,8 +40,9 @@ type AzureResource struct {
 }
 
 var AzureResourceTypes = map[string]string{
-	inventory.VirtualMachineAssetType: fetching.AzureVMType,
+	inventory.DiskAssetType:           fetching.AzureDiskType,
```

Comment:
I actually find having two constants and a map harder to read than just having a map with hardcoded strings, since the strings are only used in this map I suggest inlining these constants.